src/apps/material/material.tsx

```diff
@@ -28,6 +37,40 @@ const Material: React.FC<MaterialProps> = ({ pid, searchUrl }) => {
     return <div>No work data</div>;
   }
 
+  const {
+    manifestations,
+    titles,
+    materialTypes,
+    mainLanguages,
+    creators,
+    workYear
+  } = data.work;
+
+  const creatorsText = creatorsToString(
+    flattenCreators(filterCreators(creators, ["Person"])),
+    t
+  );
+
+  const listDescriptionData = {
+    Type: {
```

Comment:
Consider if these keys should be translatable and managed with `t()`.

I now realize that they here and elsewhere are in fact part of the user interface static texts. In that case they should be managed with `t()`.